tokenmagic/module/tokenmagic.js

```diff
@@ -1348,10 +1293,10 @@ function onMeasuredTemplateConfig(data, html) {
     // forming our injected html
     var tmfxValues;
     var selected = '';
-    tmfxValues += '<option value="NOFX">-- No FX --</option>';
+    tmfxValues += `<option value="${emptyPreset}"></option>`;
     presets.sort(compare).forEach(preset => {
         if (flag) (Magic._checkFilterId(tmTemplate, preset.name, flag) ? selected = ' selected' : selected = '');
-        tmfxValues += `<option value="TMFX_${preset.name}"${selected}>${preset.name}</option>`;

```

Comment:
I removed the prefix here to simplify applying effects in other parts of the code - there were existing bugs that didn't take this into account for new templates, and removing the prefix here removes the potential of those bugs. If there is a technical reason for including the prefix let me know and I'll adjust.

Reply:
No problem with that. I no longer know why i had done so.